config/docker-compose.yml

```diff
@@ -8,14 +8,15 @@ volumes:
 
 services:
   postgres:
-    image: "postgres@sha256:f4603c7b8aaf418393edb8cd5e2d1abd91d686ab571302dc83f887ea4a56286b"
+    image: "postgres:10.15@sha256:95f79893ef4d769277f7687b03bb24fe77a49ad2ec1c269f89a0cf849d6f4c7b"                                                                   
     ports:
       - "5432:5432"
     volumes:
       - ./sql:/docker-entrypoint-initdb.d
       - postgres-data:/var/lib/postgresql/data
     environment:
       POSTGRES_USER: $USER
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
Why is this needed?

Reply:
Argh - meant to break this out into a separate commit, but forgot. It's due to a change in the Docker container that now requires us to be explicit that we don't want a root password: https://github.com/docker-library/postgres/pull/658

Reply:
I've amended the commit description to include this. Will now merge when green